### DISCUSSION TITLE
Inflow_noise Channel

### DISCUSSION THREAD

**Discussion by rugric** [xcompact3d/Incompact3d#173] [Q&A] [answered]
Hi everyone. May I ask how and in what part of the code the parameter "inflow_noise" in the input.i3d file affected in the case of the channel-flow? Thanks for the help

**Comment by slaizet** [comment, marked as answer]
Hi,
The channel flow is periodic in the streamwise direction of the flow so inflow/outflow parameters are not relevant (inflow_noise is not used for the periodic channel flow setup).
Best,
Sylvain

**Reply by rugric** [reply]
Thank you so much!
